### DISCUSSION TITLE
Any of the latest releases support UCS2904/TM1814 IC ?

### DISCUSSION THREAD

**Discussion by tmblue** [forkineye/ESPixelStick#797] [Q&A]
I bought a bunch for perm lighting and just discovered that the protocol is an inverted  ws281x. .. ACK!! I'm been out of a game for a couple of years. So was curious if there was something here that would work.

**Comment by MartinMueller2003** [comment]
Yes some of the platforms have 2814 support in them. Or you can build your own and enable it for your platform.

**Comment by MartinMueller2003** [comment]
FYI: 2814 RGB implementations will work with the WS2811 timings.
